### PR TITLE
Update minco to 2.0.19

### PR DESCRIPTION
**Pull request by scribblemaniac**

### CODE REVIEW

File: Casks/minco.rb [hunk 1]
```diff
@@ -1,12 +1,16 @@
 cask :v1 => 'minco' do
-  version :latest
-  sha256 :no_check
+  version '2.0.19'
+  sha256 'ec81e51821620963fc8d8dc340a76143e6165e956526e8dc9bbc7c511ba5804e'
 
-  url 'http://www.celmaro.com/files/minco/Minco.zip'
-  appcast 'https://ssl.webpack.de/celmaro.com/updates/minco/minco.xml'
+  url "https://ssl.webpack.de/celmaro.com/updates/minco2/Minco#{version.gsub('.','')}.zip"
```
Comment (vitorgalvao) [inline Casks/minco.rb]: [Needs comment](https://github.com/caskroom/homebrew-cask/blob/fae625f1e35d8f1d9e5982ba92a83fcf9da87e1a/Casks/screens-connect.rb#L5) (different one from usual).

**Comment by scribblemaniac** [comment]
You've got such a good eye for details! Fixed.

**Comment by vitorgalvao** [comment]
Thank you.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as f66e06494b7c3ef5d0d15f370213d2a04ba244f6 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**